fdbserver/tester.actor.cpp

```diff
@@ -690,6 +690,8 @@ ACTOR Future<DistributedTestResults> runWorkload( Database cx, std::vector< Test
 		}
 	}
 
+	printf("success:%d failure:%d\n", success, failure);
```

Comment:
doesn't a printf already exist with success and failure?

Reply:
the existing printf only shows the binary results. It does not show how many workloads are successful.  If it is still needed to remove this printf, I can do that.